### ISSUE TITLE
[CLI] Possible typo in `app/bin/quasar-new` helper (and associated doc)

### ISSUE THREAD

**Issue by Aryess** [quasarframework/quasar#11409]
**Describe the bug**
As a fresh new user, I was going through the docs (which are great btw), and noticed that the documentation for the `new`  command of the CLI ([here](https://quasar.dev/quasar-cli/commands-list#new)) gives the formatting options as such:
```
                        Option can be:
                             * ts-options - TS options API
                             * ts-composition - TS component API
                             * ts-class - [DEPRECATED] TS class style syntax
                             * ts - use for TS boot file and store modules only
```

Shouldn't the second one say "TS **_composition_** API" instead of "component API"?

**To Reproduce**
1. Install Quasar CLI
2. Run `quasar new -h`

**Platform (please complete the following information):**
Quasar Version: 2.3.2
@quasar/app Version: 3.2.3

**Comment by rstoenescu** [comment]
Thanks for reporting!
Fix will be available in "@quasar/app" v3.2.4

**Comment by Aryess** [comment]
> Thanks for reporting! Fix will be available in "@quasar/app" v3.2.4

No worries, thanks for the fix. FYI, there's still one more in the corresponding doc: https://github.com/quasarframework/quasar/blob/0e8dc67acc0f1305372555bd2a76e69f9b242b28/docs/src/pages/quasar-cli/commands-list.md?plain=1#L388
